packages/react/test/specs/components/Dropdown/Dropdown-test.tsx

```diff
@@ -34,20 +35,169 @@ describe('Dropdown', () => {
     })
   })
 
+  describe('open', () => {
+    const onOpenChange = jest.fn()
+
+    afterEach(() => {
+      onOpenChange.mockReset()
+    })
+
+    it('is false when closed by trigger button click', () => {
+      const wrapper = mountWithProvider(<Dropdown onOpenChange={onOpenChange} items={items} />)
+      const triggerButton = wrapper.find(`button.${Dropdown.slotClassNames.triggerButton}`)
+
+      triggerButton.simulate('click').simulate('click')
+
+      expect(onOpenChange).toBeCalledTimes(2)
+      expect(onOpenChange).toHaveBeenLastCalledWith(
+        null,
+        expect.objectContaining({
+          open: false,
+        }),
+      )
+    })
```

Comment:
```
expect(onOpenChange).toHaveBeenLastCalledWithMatch(null, { open: false })
```

Not related to this PR, but I suggest to introduce a custom matcher for these cases (6 lines vs 1)...